src/cli.rs

```diff
@@ -626,10 +626,10 @@ fn init_commands<'a, C: CLI<'a>>() -> Vec<Command<'a, C>> {
                                 }
                             };
                             // Try to get a readable name for the asset.
-                            let asset = if txn.asset == AssetCode::native() {
+                            let asset = if *txn.asset() == AssetCode::native() {
                                 String::from("Native")
                             } else if let Some(asset) = keystore
-                                .asset(txn.asset)
+                                .asset(*txn.asset())
                                 .await
                             {
                                 if let Some(mint_info) = asset.mint_info() {
```

Comment:
Also unrelated to this PR, but: why are we not using `asset.name()` here? If it's what I think (that this code predates `Asset::name` and `Asset::description`) we should probably rework this a little bit. cc @shenkeyao